### ISSUE TITLE
Throws System.FormatException when reading a url

### ISSUE THREAD

**Issue by frededk** [ewoutkramer/fhir-net-api#138]
I copied the code from an example, and it used to work, but after upgrading the library with NuGet from "Version 0.90.2.33399" to "Version 0.90.3.42000" an exception is thrown.

var pat = client.Read<Patient>("Patient/1")

Throws the exception "An unhandled exception of type 'System.FormatException' occurred in mscorlib.dll"

**Comment by ewoutkramer** [comment]
Hi frededk, this is not enough info for me to reproduce your issue. Against which server did you do this request?

**Comment by frededk** [comment]
Hi ewoutkramer,

Thank you for pointing me in the direction of the server. I had been having some issues with the url and then I changed it and got another Exception (I thought it was the url again). I'm implementing my own c# server to wrap an API on another server. On the server I had added a float in the returned bundle of observations that caused all the mess. It was converted to a String. In my country we use commas instead of decimals so I had to change the CultureInfo.    

Thanks for the help!
